demo/paper-button.html

```diff
@@ -9,12 +9,16 @@
 -->
 
 <link rel="import" href="../../polymer/polymer.html">
+<!-- required for 1.0 version of demo -->
 <link rel="import" href="../../paper-material/paper-material.html">
```

Comment:
`paper-styles/element-styles/paper-material` is back ported to 1.0, so this shouldn't be needed